src/seeder/seeder.js

```diff
@@ -1,24 +1,121 @@
-const { upsertObject } = require('../connection/con');
+const { cluster,upsertObject,getObject } = require('../connection/con');
 const { falshMessage } = require('../dispatcher/responseDispatcher');
-// const { gameVariable } = require('./gameData')
+const { DBDocType } = require('../configuration/constants');
+const { mapper } = require('../mapper/game');
 
 class Seeder {
 
     /**
-     * static data of game
+     * static data of game and store data in data base 
+     * check if game all ready store in data base then send error response
      * @param {Request} req 
      * @param {response} res 
      */
-    seedGameObject(req,res){
+    addGameObject(req,res){
         const gameVariable = req.body;
-        upsertObject(req.body.gameName,gameVariable).then((result) =>{
-            let response = falshMessage.resDispatch(res,'GAME_DATA',{gameVariable});
+        getObject(req.body.gameName).then((reslt) =>{
+            if(reslt){
+                let response = falshMessage.resDispatchError(res,'GAME_EXISTS');
+                return response;
+            }
+        }).catch(error =>{
+            let docType = DBDocType.GAME;
```

Comment:
why these variables are declared with `let` keyword?? Aren't they supposed to be `const` ??